naga/src/back/msl/writer.rs

```diff
@@ -55,6 +55,7 @@ pub(crate) const MODF_FUNCTION: &str = "naga_modf";
 pub(crate) const FREXP_FUNCTION: &str = "naga_frexp";
 pub(crate) const ABS_FUNCTION: &str = "naga_abs";
 pub(crate) const DIV_FUNCTION: &str = "naga_div";
+pub(crate) const DOT_FUNCTION: &str = "naga_dot";
```

Comment:
Calling this `DOT_FUNCTION` is potentially confusing, because it's not actually the name of the function in the sense that the rest of these are. Maybe `DOT_FUNCTION_PREFIX`?